OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -279,7 +279,7 @@ public bool CanMoveFreelyInto(Actor actor, CPos cell, Actor ignoreActor, CellCon
 				return true;
 
 			// Cache doesn't account for ignored actors - these must use the slow path
```

Comment:
Imho the comment should be updated with a reference to temporary blockers, like `Cache doesn't account for ignored actors or temporary blockers - these must use the slow path`.